Tests/MongoSwiftSyncTests/UnifiedTestRunner/HasListableProperties.swift

```diff
@@ -0,0 +1,21 @@
+import MongoSwiftSync
+
+protocol HasListableProperties {
```

Comment:
there doesn't seem to be a way to extend an arbitrary object. In retrospect I guess I could have added a top level function which took in `Any` but this feels a little more idiomatic and readable